### ISSUE TITLE
[Docs Bug] npm webhint (hint) page: 404s 

### ISSUE THREAD

**Issue by henrihelvetica** [webhintio/hint#3792]
# 📚 npm webhint (hint) Quick start user guide page: 404s 

Perusing the [NPM page](https://www.npmjs.com/package/hint) and several links are broken. 

* **troubleshoot section**: https://github.com/webhintio/hint/blob/HEAD/docs/user-guide/troubleshoot/summary.md
* **Configurations**:  https://github.com/webhintio/hint/blob/HEAD/docs/user-guide/concepts/configurations.md
* **Hints**:  https://github.com/webhintio/hint/blob/HEAD/docs/user-guide/concepts/hints.md
* **Connectors**: https://github.com/webhintio/hint/blob/HEAD/docs/user-guide/concepts/connectors.md
* **Formatters**:  https://github.com/webhintio/hint/blob/HEAD/docs/user-guide/concepts/formatters.md
* **Parsers**:  https://github.com/webhintio/hint/blob/HEAD/docs/user-guide/concepts/parsers.md
* **Local version**:  https://github.com/webhintio/hint/blob/HEAD/docs/contributor-guide/index.md

**Comment by molant** [comment]
This is because we optimized for local links when publishing so you open the local version. Looks like there is an issue between npm an mono repos when they create the links automatically.

Options:
- Research if there's a way to link to a specific folder in `package.json` and not just a repo (this will be the ideal scenario)
- Make a not for those reading from npm that they should go to the website to make sure all links are working and have access to more information
- Update those links to point to the right versions of the website. This could maybe mess with the link transformation step we have with the website

**Comment by hxlnt** [comment]
Like this? From https://docs.npmjs.com/files/package.json#repository:

If the package.json for your package is not in the root directory (for example if it is part of a monorepo), you can specify the directory in which it lives:
```
"repository": {
  "type" : "git",
  "url" : "https://github.com/facebook/react.git",
  "directory": "packages/react-dom"
}
```

**Comment by molant** [comment]
Yes, like this. I believe we tried that a while ago but wasn't working but maybe we should try again?

**Comment by hxlnt** [comment]
@johnemau Tony thinks we already implement this fix above, and that it's possible this may be a bug on npm. Do you mind taking a look?

**Comment by johnemau** [comment]
> Update those links to point to the right versions of the website. This could maybe mess with the link transformation step we have with the website

@molant, after researching the root cause in npm (https://github.com/npm/marky-markdown/issues/449) I went with this fix (#3937) from your [comment](https://github.com/webhintio/hint/issues/3792#issuecomment-637076471).

Can you clarify how this might _"mess with the link transformation step"_ ? How can I check if this is the case?

**Comment by molant** [comment]
Good question. Before we were cloning the latest version of master in webhint.io, but now we are pulling the latest npm packages.

I think that a way would be clone the website's repo locally, install all dependencies, manually update a couple of the files you've modified, and then run the generation scripts to see what comes out of that.

**Comment by johnemau** [comment]
> I think that a way would be clone the website's repo locally, install all dependencies, manually update a couple of the files you've modified, and then run the generation scripts to see what comes out of that.

@molant, tested it out after https://github.com/webhintio/webhint.io/pull/934, and my fix worked #3967
